api/v1beta1/cluster_types.go

```diff
@@ -1047,7 +1047,9 @@ type Cluster struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   ClusterSpec   `json:"spec,omitempty"`
+	// spec is the desired state of Cluster
```

Comment:
```suggestion
	// spec is the desired state of Cluster.
```

Reply:
Thanks for the review! Fixed with at https://github.com/kubernetes-sigs/cluster-api/pull/11870/commits/1281b4babc9e5370b8e41ca19f3577eb174860a9